timesketch/api/v1/resources/searchtemplate.py

```diff
@@ -84,7 +84,7 @@ class SearchTemplateParseResource(resources.ResourceMixin, Resource):
     """Resource to parse a search template query string using Jinja2 template."""
 
     @login_required
-    def get(self, searchtemplate_id):
+    def post(self, searchtemplate_id):
         """Parse the query string template with Jinja2.
 
         This resource take a form with parameters. These will be sent to the Jinja2
```

Comment:
there is a `will` missing

Reply:
I don't understand what you mean with `will missing` :) But I rewording this sentence to make it more accurate.